### PR TITLE
[Release-1.0] Relax prometheus cr operator test

### PR DESCRIPTION
**Pull request by assafad**
Manual cherry-pick of https://github.com/kubevirt/kubevirt/pull/10238.

**Release note**:

```release-note
None
```

### CODE REVIEW

File: tests/operator/operator.go [hunk 1]
```diff
@@ -2527,12 +2527,9 @@ spec:
 			monv1 := virtClient.PrometheusClient().MonitoringV1()
 			prometheusRule, err := monv1.PrometheusRules(flags.KubeVirtInstallNamespace).Get(context.Background(), components.KUBEVIRT_PROMETHEUS_RULE_NAME, metav1.GetOptions{})
 			Expect(err).ShouldNot(HaveOccurred())
-			hasWorkloadUpdates := false
-			if len(originalKv.Spec.WorkloadUpdateStrategy.WorkloadUpdateMethods) > 0 {
-				hasWorkloadUpdates = true
-			}
```
Comment (0xFelix) [inline tests/operator/operator.go:2533]: Please keep these lines, they were removed in main as part of https://github.com/kubevirt/kubevirt/commit/61a98f1267b0a8cfb8d879fa296c4c75556dfb4b but this was not backported.

Comment (assafad) [reply tests/operator/operator.go:2533]: But `hasWorkloadUpdates` isn't in use after removing `NewPrometheusRuleSpec()`.

Comment (0xFelix) [reply tests/operator/operator.go:2533]: You are right, never mind.

**Review by 0xFelix — COMMENTED** [review]
Please keep the original commit message/title and put a NOTE and the conflicting files into the commit message.

**Review by xpivarc — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**